apps/rush-lib/src/api/CommandLineConfiguration.ts

```diff
@@ -368,11 +584,7 @@ export class CommandLineConfiguration {
     this._additionalPathFolders.unshift(pathFolder);
   }
 
-  public get shellCommandTokenContext(): Readonly<IShellCommandTokenContext> | undefined {
-    return this._shellCommandTokenContext;
-  }
-
-  public set shellCommandTokenContext(tokenContext: IShellCommandTokenContext | undefined) {
-    this._shellCommandTokenContext = tokenContext;
+  private _normalizeNameForLogFilenameIdentifiers(name: string): string {
+    return name.replace(/:/g, '_'); // Replace colons with underscores to be filesystem-safe
```

Comment:
Add a comment explaining that ts-command-line restricts command names to lowercase letters, numbers, and underscores.  Without this important constraint, the RegExp would not produce a filesystem-safe output.

Reply:
Looks like we are also calling `_normalizeNameForLogFilenameIdentifiers()` using phase names.  Double-check that this constraint applies to them as well.